src/components/views/Homepage/Homepage.js

```diff
@@ -5,11 +5,12 @@ import styles from './Homepage.module.scss';
 
 import FeatureBoxes from '../../features/FeatureBoxes/FeatureBoxes';
 import NewFurniture from '../../features/NewFurniture/NewFurnitureContainer';
-import BrandMarks from '../../features/BrandMarks/BrandMarks';
```

Comment:
To raczej było potrzebne

Reply:
to dotyczy drugiego komponentu Brand (zostało usunięte)